changelog.txt

```diff
@@ -2,6 +2,12 @@
 
 Version 1.9.11, 1 February 2016
 
+The Checker Framework distribution now contains annotated versions
```

Comment:
I have updated the text and I think it's OK.  The lib/ directory does have both version 5.2 and 6.0 pre-release.